MAUI/Popup/Popup-Events.md

```diff
@@ -0,0 +1,285 @@
+{%endhighlight%}
+
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="4 9" %}
+public MainPage()
+{
+    InitializeComponent();
+    popup.Opening += Popup_Opening;
+}
+
+private void Popup_Opening(object sender, System.ComponentModel.CancelEventArgs e)
+{
+    e.Cancel = true;
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Opened event
+
+The `SfPopup.Opened` event will be fired whenever displaying the Popup in the application.
+
+You can execute your own set of codes once the popup is opened, and visible in the application in its respective event handler.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<sfPopup:SfPopup x:Name="popup" Opened="popup_Opened"/>
+{%endhighlight%}
+
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="4" %}
+public MainPage()
+{
+    InitializeComponent();
+    popup.Opened += Popup_Opened;
+}
+
+private void Popup_Opened(object sender, EventArgs e)
+{
+   // Codes that needs to be executed once popup is visible in the screen.
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Closing event
+
+The `SfPopup.Closing` event will be fired whenever closing the Popup in the application. It can cancel popup closing with `CancelEventArgs` that contains the following property:
+
+* `Cancel`: Popup opening is based on this value.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<sfPopup:SfPopup x:Name="popup"  Closing="Popup_Closing"/>
+{%endhighlight%}
+
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="4 9" %}
+public MainPage()
+{
+    InitializeComponent();
+    popup.Closing += Popup_Closing;
+}
+
+private void Popup_Closing(object sender, System.ComponentModel.CancelEventArgs e)
+{
+    e.Cancel = true;
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Closed event
+
+The `SfPopup.Closed` event will be fired whenever dismissing the Popup from the view.
+
+You can execute your own set of codes once the popup is completely closed in its respective event handler.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<sfPopup:SfPopup x:Name="popup" Closed="Popup_Closed"/>
+{%endhighlight%}
+
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="4" %}
+public MainPage()
+{
+    InitializeComponent();
+    popup.Closed += Popup_Closed;
+}
+
+private void Popup_Closed(object sender, EventArgs e)
+{
+    // Codes that needs to be executed once popup is completely closed.
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Accept command
+
+The `SfPopup.AcceptCommand` will be fired when clicking the Accept button in the popup footer.
+
+To handle the Accept button,
+
+* Derive a class from `ICommand`, and implement the `ICommand` interface.
+* To prevent popup from closing, return false in the `CanExecute()` override method and the `Execute()` override method will not be fired.
+* Else return true in the `CanExecute()` override method and do the required operations in the `Execute()` method.
+* Now create a property of your custom command type in the view model class and initialize it.
```

Comment:
* Now, create a property of your custom command type in the view model class and initialize it.